commit 4c736420952f355f18bdc4f4ea2d16e4514fa034
Author: Takuto Ikuta <tikuta@chromium.org>
Date:   Fri May 2 06:29:25 2025 -0700

    content: Fix missing headers in `opener_heuristic_utils.h`.
    
    Some include were missing from the header which could lead to compilation errors.
    
    Bug: 40263312
    Change-Id: I113af598e6cd5a7fa05a8605e4fa3e105e0a777b
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/6495605
    Commit-Queue: Joshua Hood <jdh@chromium.org>
    Auto-Submit: Takuto Ikuta <tikuta@chromium.org>
    Reviewed-by: Joshua Hood <jdh@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1454945}

diff --git a/content/browser/tpcd_heuristics/opener_heuristic_utils.h b/content/browser/tpcd_heuristics/opener_heuristic_utils.h
index f4c344420f489..b02761cf61da6 100644
--- a/content/browser/tpcd_heuristics/opener_heuristic_utils.h
+++ b/content/browser/tpcd_heuristics/opener_heuristic_utils.h
@@ -6,6 +6,9 @@
 #define CONTENT_BROWSER_TPCD_HEURISTICS_OPENER_HEURISTIC_UTILS_H_
 
 #include <map>
+#include <set>
+#include <string>
+#include <utility>
 
 #include "base/types/optional_ref.h"
 #include "content/common/content_export.h"
